From acd175aa53af1f1b8422bf1f1fe57eafc710cdca Mon Sep 17 00:00:00 2001
From: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Date: Fri, 2 Dec 2016 23:09:44 +0100
Subject: [PATCH] Makefile: allow passing CFLAGS/LDFLAGS

This might be needed to pass some custom CFLAGS/LDFLAGS when building
rpiboot.

Submitted-upstream: https://github.com/raspberrypi/usbboot/pull/2
Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
[Rebased on 9324fd7]
Signed-off-by: Peter Seiderer <ps.report@gmx.net>
[Rebased on 20221215-105525-0-g1afa26c5]
Signed-off-by: Fiona Klute <fiona.klute+wiwa@gmx.de>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index dc78cfe..c9d4910 100755
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 PKG_VER=$(shell grep rpiboot debian/changelog | head -n1 | sed 's/.*(\(.*\)).*/\1/g')
 GIT_VER=$(shell git rev-parse HEAD 2>/dev/null | cut -c1-8 || echo "")
 rpiboot: main.c msd/bootcode.h msd/start.h msd/bootcode4.h msd/start4.h
-	$(CC) -Wall -Wextra -g -o $@ $< `pkg-config --cflags --libs libusb-1.0` -DGIT_VER="\"$(GIT_VER)\"" -DPKG_VER="\"$(PKG_VER)\""
+	$(CC) -Wall -Wextra -g $(CFLAGS) -o $@ $< `pkg-config --cflags --libs libusb-1.0` -DGIT_VER="\"$(GIT_VER)\"" -DPKG_VER="\"$(PKG_VER)\"" $(LDFLAGS)
 
 %.h: %.bin ./bin2c
 	./bin2c $< $@
-- 
2.45.2

